yarn.lock

```diff
@@ -5548,6 +5562,11 @@ wcwidth@^1.0.1:
   dependencies:
     defaults "^1.0.3"
 
+web-streams-polyfill@^3.0.3:
```

Comment:
is still this needed ? I think Node now supports web streams

Reply:
it's only conditionally being used on env that dose not have it, and fetch-blob supports older versions before it was supported